packages/shared/src/components/post/PostModalActions.tsx

```diff
@@ -79,7 +79,7 @@ export function PostModalActions({
 
   return (
     <Container {...props} className={classNames('gap-2', className)}>
-      {onReadArticle && (
+      {onReadArticle && post?.type !== 'welcome' && (
```

Comment:
Not sure this will be the most solid fix.
Is there anyway we can not add onReadArticle if the post is welcome?

Asking this because soon we'll introduce more types that don't have clicks, so we need to have a more generic way down the line.

Reply:
I agree, this isn't very scalable, I have made changes to use the onReadArticle to try and make it more scalable now